scripts/gis-material-update/config.yaml

```diff
@@ -62,6 +62,11 @@ cycle_infra:
   target_buffer_file: "tormays_cycle_infra_polys.gpkg"
   buffer:
     - 20
+central_business_area:
+  addr: "WFS:https://kartta.hel.fi/ws/geoserver/avoindata/wfs"
+  layer: "avoindata:Piirijako_pienalue"
```

Comment:
Is there a layer for osaalue? That's the lowest level we care about.

Reply:
I'll answer myself: Yes there is, but it doesn't have peruspiiri data, so we can't filter it.

Reply:
Osaalue(=osaalue_nimi_fi) and peruspiiri(=peruspiiri_nimi_fi) are tormays_central_business_areas attribute fields